### PR TITLE
Updated cstor overiew dashboard

### PR DESCRIPTION
**Pull request by Ab-hishek**
This PR makes a few changes in the cstor-overview dashboard. The change include:
1. namespace variable changed to display namespaces where openebs is installed instead of having a drop-down to display all namespaces inside the cluster.
2. datasource was not be used by the individual panels. Changes are made so that this variable is used by the panels.
3. datasource value was hard-coded. Changed it and made it configurable via variables in grafana.

### CODE REVIEW

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json [hunk 1]
```diff
@@ -538,7 +522,7 @@
         },
         {
           "exemplar": true,
-          "expr": "sum(container_memory_usage_bytes{namespace=\"openebs\"} * on (name,pod)group_left(cstor_pool,namespace) openebs_pool_status) by (cstor_pool)",
+          "expr": "sum(container_memory_usage_bytes{namespace=\"$openebs_namespace\"} * on (name,pod)group_left(cstor_pool,namespace) openebs_pool_status) by (cstor_pool)",
```
Comment (avishnu) [inline deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json:525]: good catch

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json [hunk 2]
```diff
@@ -833,19 +810,18 @@
...
-        "definition": "query_result(sum(container_memory_working_set_bytes{namespace!=\"\"}) by (namespace))",
+        "datasource": "$datasource",
+        "definition": "label_values(openebs_pool_status, namespace)",
         "description": null,
         "error": null,
         "hide": 0,
-        "includeAll": true,
-        "label": "Namespace",
+        "includeAll": false,
+        "label": "openebs namespace",
```
Comment (avishnu) [inline deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json]: Should it be OpenEBS namespace?

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json [hunk 3]
```diff
@@ -833,19 +810,18 @@
       {
         "allValue": null,
         "current": {},
-        "datasource": null,
-        "definition": "query_result(sum(container_memory_working_set_bytes{namespace!=\"\"}) by (namespace))",
+        "datasource": "$datasource",
+        "definition": "label_values(openebs_pool_status, namespace)",
```
Comment (avishnu) [inline deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json:814]: Why this change?

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json [hunk 4]
```diff
@@ -833,19 +810,18 @@
...
         "description": null,
         "error": null,
         "hide": 0,
-        "includeAll": true,
-        "label": "Namespace",
+        "includeAll": false,
```
Comment (avishnu) [inline deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json:818]: why this change?

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json (hunk 2 above)
Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json]: Okay. Will change it.

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json (hunk 3 above)
Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json:814]: This will list only the namespaces where openebs is installed because cstor pools are always formed in the openebs installed ns and using this query we can get list of only OpenEBS ns. Earlier query would give us all the ns in the cluster which we do not want.

File: deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json (hunk 4 above)
Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/dashboards/cStor/cStor-overview.json:818]: Earlier All option was included just so that namespace variable doesn't randomly choose any ns and show in the dashboard. That would have given the user the feeling that OpenEBS is installed in that ns. That's why all option was added. But here we get only the OpenEBS ns therefore this can be omitted.

**Review by avishnu — APPROVED** [review]
lgtm

**Review by rajaSahil — APPROVED** [review]

**Review by avishnu — APPROVED** [review]
Lgtm
